Source/Core/DolphinQt/Settings/AudioPane.cpp

```diff
@@ -160,26 +160,28 @@ void AudioPane::CreateWidgets()
   auto* playback_layout = new QGridLayout;
   playback_box->setLayout(playback_layout);
 
-  ConfigSlider* audio_buffer_size = new ConfigSlider(16, 512, Config::MAIN_AUDIO_BUFFER_SIZE, 8);
+  m_audio_buffer_size = new ConfigSlider(16, 512, Config::MAIN_AUDIO_BUFFER_SIZE, 8);
   QLabel* audio_buffer_size_label = new QLabel;
 
-  audio_buffer_size->setSingleStep(8);
-  audio_buffer_size->setPageStep(8);
+  m_audio_buffer_size->setSingleStep(8);
+  m_audio_buffer_size->setPageStep(8);
 
-  audio_buffer_size->SetDescription(
-      tr("Controls the number of audio samples buffered."
-         " Lower values reduce latency but may cause more crackling or stuttering."
-         "<br><br><dolphin_emphasis>If unsure, set this to 80 ms.</dolphin_emphasis>"));
+  m_audio_buffer_size->SetDescription(
+      tr("Controls the number of audio samples buffered. "
+         "Lower values reduce latency but may cause more crackling or stuttering."
+         "<br><br>LLE Requires larger buffers than HLE because it emits audio in larger blocks."
+         "<br><br><dolphin_emphasis>If unsure, set this to 80 ms for HLE or 120 ms for "
+         "LLE</dolphin_emphasis>"));
 
   // Connect the slider to update the value label live
-  connect(audio_buffer_size, &QSlider::valueChanged, this, [=](int value) {
+  connect(m_audio_buffer_size, &QSlider::valueChanged, this, [=](int value) {
```

Comment:
```suggestion
  connect(m_audio_buffer_size, &QSlider::valueChanged, this, [audio_buffer_size_label, this](int value) {
```
Using `m_audio_buffer_size` introduced a warning: `Implicit capture of 'this' with a capture default of '=' is deprecated clang(-Wdeprecated-this-capture)`.